### PR TITLE
make "systemd-dissect --mount" available as /sbin/mount.ddi

### PR DESCRIPTION
**Pull request by poettering**
This way, people can mount DDIs via /etc/fstab. Yay!

### CODE REVIEW

File: man/systemd-dissect.xml [hunk 1]
```diff
@@ -359,14 +368,43 @@
...
 
+  <refsect1>
+    <title>Invocation as <command>/sbin/mount.ddi</command></title>
+
+    <para>The <command>systemd-dissect</command> executable may be symlinked to
+    <filename>/sbin/mount.ddi</filename>. If invoked through that it implements <citerefentry
+    project='man-pages'><refentrytitle>mount</refentrytitle><manvolnum>8</manvolnum></citerefentry>'s
+    "external helper" interface for the (pseudo) file system type <literal>ddi</literal>. This means
+    conformat disk images may be mounted directly via</para>
+
```
Comment (mrc0mmand) [inline man/systemd-dissect.xml]:
```suggestion
    conformant disk images may be mounted directly via</para>

```

**Review by bluca — APPROVED** [review]

**Review by yuwata — APPROVED** [review]
Sounds nice! Not necessary in this PR, but may be better to also test a fstab entry.

**Comment by yuwata** [comment]
Ubuntu CI fails:
```
[  135.498946] testsuite-50.sh[358]: + mount -t ddi /tmp/tmp.XFx3pA/minimal_0.gpt /tmp/mounthelper.7991 -o ro,X-mount.mkdir,discard
[  135.658547] testsuite-50.sh[1066]: mount: /tmp/mounthelper.7991: unknown filesystem type 'ddi'.
```
I guess the install path is wrong.

**Comment by yuwata** [comment]
Not familiar with deb package building, but mount.ddi is not packaged. So, the test environment does not have it.
```
make[1]: Entering directory '/tmp/autopkgtest.c1l4OI/build.vvf/systemd'
dh_missing --list-missing
dh_missing: warning: sbin/mount.ddi exists in debian/tmp but is not installed to anywhere 
	The following debhelper tools have reported what they installed (with files per package)
	 * dh_install: libnss-myhostname (3), libnss-mymachines (3), libnss-resolve (3), libnss-systemd (3), libpam-systemd (3), libsystemd-dev (700), libsystemd0 (2), libudev-dev (72), libudev1 (2), libudev1-udeb (2), systemd (43), systemd-container (50), systemd-coredump (16), systemd-journal-remote (24), systemd-oomd (19), systemd-standalone-sysusers (0), systemd-standalone-tmpfiles (0), systemd-sysv (14), systemd-tests (1), systemd-timesyncd (12), udev (42), udev-udeb (22)
	 * dh_installdocs: libnss-myhostname (0), libnss-mymachines (0), libnss-resolve (0), libnss-systemd (0), libpam-systemd (0), libsystemd-dev (0), libsystemd0 (0), libudev-dev (0), libudev1 (0), systemd (0), systemd-container (0), systemd-coredump (0), systemd-journal-remote (0), systemd-oomd (0), systemd-standalone-sysusers (0), systemd-standalone-tmpfiles (0), systemd-sysv (0), systemd-tests (0), systemd-timesyncd (0), udev (0)
	 * dh_installman: libnss-myhostname (0), libnss-mymachines (0), libnss-resolve (0), libnss-systemd (0), libpam-systemd (0), libsystemd-dev (0), libsystemd0 (0), libudev-dev (0), libudev1 (0), systemd (0), systemd-container (0), systemd-coredump (0), systemd-journal-remote (0), systemd-oomd (0), systemd-standalone-sysusers (0), systemd-standalone-tmpfiles (0), systemd-sysv (0), systemd-tests (0), systemd-timesyncd (0), udev (0)
	If the missing files are installed by another tool, please file a bug against it.
	When filing the report, if the tool is not part of debhelper itself, please reference the
	"Logging helpers and dh_missing" section from the "PROGRAMMING" guide for debhelper (10.6.3+).
	  (in the debhelper package: /usr/share/doc/debhelper/PROGRAMMING.gz)
	Be sure to test with dpkg-buildpackage -A/-B as the results may vary when only a subset is built
	If the omission is intentional or no other helper can take care of this consider adding the
	paths to debian/not-installed.
make[1]: Leaving directory '/tmp/autopkgtest.c1l4OI/build.vvf/systemd'
```

**Comment by bluca** [comment]
> Not familiar with deb package building, but mount.ddi is not packaged. So, the test environment does not have it.
> 
> ```
> make[1]: Entering directory '/tmp/autopkgtest.c1l4OI/build.vvf/systemd'
> dh_missing --list-missing
> dh_missing: warning: sbin/mount.ddi exists in debian/tmp but is not installed to anywhere 
> 	The following debhelper tools have reported what they installed (with files per package)
> 	 * dh_install: libnss-myhostname (3), libnss-mymachines (3), libnss-resolve (3), libnss-systemd (3), libpam-systemd (3), libsystemd-dev (700), libsystemd0 (2), libudev-dev (72), libudev1 (2), libudev1-udeb (2), systemd (43), systemd-container (50), systemd-coredump (16), systemd-journal-remote (24), systemd-oomd (19), systemd-standalone-sysusers (0), systemd-standalone-tmpfiles (0), systemd-sysv (14), systemd-tests (1), systemd-timesyncd (12), udev (42), udev-udeb (22)
> 	 * dh_installdocs: libnss-myhostname (0), libnss-mymachines (0), libnss-resolve (0), libnss-systemd (0), libpam-systemd (0), libsystemd-dev (0), libsystemd0 (0), libudev-dev (0), libudev1 (0), systemd (0), systemd-container (0), systemd-coredump (0), systemd-journal-remote (0), systemd-oomd (0), systemd-standalone-sysusers (0), systemd-standalone-tmpfiles (0), systemd-sysv (0), systemd-tests (0), systemd-timesyncd (0), udev (0)
> 	 * dh_installman: libnss-myhostname (0), libnss-mymachines (0), libnss-resolve (0), libnss-systemd (0), libpam-systemd (0), libsystemd-dev (0), libsystemd0 (0), libudev-dev (0), libudev1 (0), systemd (0), systemd-container (0), systemd-coredump (0), systemd-journal-remote (0), systemd-oomd (0), systemd-standalone-sysusers (0), systemd-standalone-tmpfiles (0), systemd-sysv (0), systemd-tests (0), systemd-timesyncd (0), udev (0)
> 	If the missing files are installed by another tool, please file a bug against it.
> 	When filing the report, if the tool is not part of debhelper itself, please reference the
> 	"Logging helpers and dh_missing" section from the "PROGRAMMING" guide for debhelper (10.6.3+).
> 	  (in the debhelper package: /usr/share/doc/debhelper/PROGRAMMING.gz)
> 	Be sure to test with dpkg-buildpackage -A/-B as the results may vary when only a subset is built
> 	If the omission is intentional or no other helper can take care of this consider adding the
> 	paths to debian/not-installed.
> make[1]: Leaving directory '/tmp/autopkgtest.c1l4OI/build.vvf/systemd'
> ```

added and restarted

**Pull request merged**